pandas/tests/io/excel/test_writers.py

```diff
@@ -838,6 +838,18 @@ def test_to_excel_multiindex_no_write_index(self, path):
         # Test that it is the same as the initial frame.
         tm.assert_frame_equal(frame1, frame3)
 
+    def test_to_excel_empty_multiindex(self, path):
```

Comment:
The gh reference has to stay (see my first comment about this)

Reply:
😂 done (I do not think the test errors were caused by my changes)